control-plane/agents/src/bin/core/volume/operations.rs

```diff
@@ -274,19 +274,20 @@ impl ResourcePublishing for OperationGuardArc<VolumeSpec> {
         let nexus_node = get_volume_target_node(registry, &state, request).await?;
         let nexus_id = NexusId::new();
 
-        let operation =
-            VolumeOperation::Publish((nexus_node.clone(), nexus_id.clone(), request.share));
+        let operation = VolumeOperation::Publish((
+            nexus_node.clone(),
+            nexus_id.clone(),
+            request.share,
+            request.republish(),
+        ));
         let spec_clone = self.start_update(registry, &state, operation).await?;
-
```

Comment:
keep the newlines to separate the `start_update` and `validate_update_step` from the rest

Reply:
Oops, will add it back.